### ISSUE TITLE
kmcp search is stucked

### ISSUE THREAD

**Issue by aababc1** [shenwei356/kmcp#43]
Hi. Thank you for your nice tool. 

I am currently searching metagenome against viral genome database that you provide in the link : https://bioinf.shenwei.me/kmcp/database/

I installed this tool using conda.
But I don't know why the search step takes too long. 
Host OS is centos 7.10 (  3.10.0-1160.el7.x86_64 )
<img width="761" alt="image" src="https://github.com/shenwei356/kmcp/assets/74400447/a1fb69a6-0b27-4261-9cff-7adc807e165f">

Thank you in advance.

**Comment by shenwei356** [comment]
Try push the  "enter" and see if another log shows up. If it does, it's running. If your input file are huge, it would take a long time.

**Comment by aababc1** [comment]
Thank you for your quick reply.
I'm sorry there was an error in my expression . 
It's not stopped, but  I think the speed is way slower  compared to your paper's  benchmark results or other user. 

The input data is about 5Gbps , illumina short paired end reads,  server has 2TB memory and 2 * Xeon 8000 series 40cpus . 

I tried compiled version and conda version  also with/without  -w option to run  KMCP . 

Used database is genbank-viral that is one of KMCP premade database. 

07:05:50.199 [INFO] processed queries: 30374219, speed: 0.159 million queries per minute
07:05:50.199 [INFO] 1.6593% (504003/30374219) queries matched
07:05:50.199 [INFO] done searching
07:05:50.199 [INFO] search results saved to: test2
07:05:50.199 [INFO] 
07:05:50.199 [INFO] elapsed time: 3h11m18.24440856s


In my guess, KMCP could analyze faster than kraken2, but in some reason it's speed is slow. 

I would like to get help, how I  can boost speed. 

downloaded version : KMCP v0.9.4

command I ran :

 ../kmcp/kmcp/kmcp search -d genbank-viral.kmcp -1 ERR1018185_fp_hg38_1.fastq.gz -2 ERR1018185_fp_hg38_2.fastq.gz -o test2  -j 20 -w 
03:54:31.955 [INFO] kmcp v0.9.4
03:54:31.955 [INFO]   https://github.com/shenwei356/kmcp
